src/domain/Exercise.java

```diff
@@ -1,7 +1,29 @@
 package domain;
 
+/**
+ * Клас що являє собою завдання
+ * @author Vlad
+ */
 public class Exercise {
-    public static float Calculate(int R){
-        return (float) (2*Math.PI*Math.sqrt(R));
+    
+    /**
+     * Визначає чи є рядок паліндромом
+     * @param text вхідний текст
+     * @return логічне значення
+     */
+    public static boolean Calculate(String text) {

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.